app/notifications/validators.py

```diff
@@ -108,16 +107,12 @@ def service_can_send_to_recipient(send_to, key_type, service, allow_whitelisted_
         raise BadRequestError(message=message)
 
 
+# TODO clean up and remove
 def service_has_permission(notify_type, permissions):
     return notify_type in [p.permission for p in permissions]
 
 
-def check_service_has_permission(notify_type, permissions):
-    if not service_has_permission(notify_type, permissions):
-        raise BadRequestError(message="Service is not allowed to send {}".format(
-            get_public_notify_type_text(notify_type, plural=True)))
-
-
+# TODO clean up and remove
```

Comment:
Please create a ticket for this TODO and add the ticket number to this comment.

Reply:
https://github.com/department-of-veterans-affairs/notification-api/issues/1410

Reply:
Please add ticket number to the comment itself
`# TODO 1410 clean up and remove.`

Reply:
done